js/bitmex.js

```diff
@@ -857,16 +862,13 @@ module.exports = class bitmex extends Exchange {
         //         "timestamp":null  # ←---------------------------- null
         //     }
         //
-        const id = this.safeString (item, 'transactID');
         const account = this.safeString (item, 'account');
-        const referenceId = this.safeString (item, 'tx');
         const referenceAccount = undefined;
-        const type = this.parseLedgerEntryType (this.safeString (item, 'transactType'));
         const currencyId = this.safeString (item, 'currency');
         const code = this.safeCurrencyCode (currencyId, currency);
-        let amount = this.safeNumber (item, 'amount');
+        let amount = this.safeString (item, 'amount');
         if (amount !== undefined) {
-            amount = amount / 100000000;
+            amount = Precise.stringDiv (amount, '100000000');
```

Comment:
@samgermain The scale differs from currency to currency, so I don't think we can blindly divide by the same value, example: 
![image](https://user-images.githubusercontent.com/43336371/189924320-6f8e27a7-a12e-484a-a945-b9ec5cc350be.png)

Reply:
I'm not sure what else to do. Those we're the numbers that we're in there before, I was just trying to convert them to string math here

Reply:
@samgermain I'm just saying I'm not sure it's worth to "string math" a flawed implementation

Reply:
It's inconsistent not to use string math. I don't understand why it's flawed, I see the arrow pointing at some numbers, but I don't know why those numbers are significant

Reply:
@samgermain Look, since the precision is wrong, it's saying the operation used 0.00016 USDT but in reality it used 16 USDT or something like that, I'm not saying your fix is wrong, but the overall implementation

Reply:
Regardless, we need to change it to string math or the user will experience errors related to dividing different types.

I would like to fix the implementation, but I don't know what the solutions would be. 

Without fixing the implementation though, we still need to use string math

Reply:
It looks like this problem has already been identified with a solution on [this PR](https://github.com/ccxt/ccxt/pull/15311). @carlosmiei  In the future could you point me in the direction of things like this if it rings a bell for you? It looks like you were part of the conversation on that PR. I spent a lot of time trying to figure this out, but I understand if you didn't recognize it as the same problem

Reply:
@carlosmiei It looks like this problem might not get fixed for a while, and meanwhile, we are preventing users from using string math with bitmex while this pr remains unmerged. Whether this PR gets merged or doesn't get merged, that problem will remain anyway so there's no reason to put this PR on hold until that other problem gets a solution